packages/datadog-plugin-oracledb/test/index.spec.js

```diff
@@ -327,21 +377,23 @@ describe('Plugin', () => {
             {
               v0: {
                 opName: 'oracle.query',
-                serviceName: 'oracledb:1521/xepdb1'
+                serviceName: config.connectString,
```

Comment:
Nit: if the service function doesn't work properly and `connAttrs.connectString` is undefined, then this will match. Checking for a hard-coded value as before avoids this issue.

Reply:
I added a check for the string being passed through correctly.
It would otherwise not work between local / CI.